snuba/datasets/storages/metrics.py

```diff
@@ -62,6 +63,17 @@
     ),
 )
 
+if settings.ENABLE_METRICS_SUBSCRIPTIONS:
```

Comment:
No need to make a difference here. Just change the existing kafka stream loader, metrics are not in prod, no need to maintain two ways